lte/protos/sms_orc8r.proto

```diff
@@ -52,7 +51,21 @@ message MessageList {
 
 // smsd-cloud service definition
 service SmsD {
-	rpc ReportDelivery(SMOUplinkUnitdata) returns (magma.orc8r.Void) {}
+	rpc ReportDelivery(ReportDeliveryRequest) returns (ReportDeliveryResponse) {}
 
-	rpc GetMessages(magma.lte.SubscriberIDSet) returns (MessageList) {}
+    rpc GetMessages(GetMessagesRequest) returns (GetMessagesResponse) {}
+}
+
+message ReportDeliveryResponse {}
+
+message ReportDeliveryRequest {
+    SMOUplinkUnitdata report = 1;
+}
+
+message GetMessagesRequest {
+    repeated string imsis = 1;
+}
+
+message GetMessagesResponse {
```

Comment:
This type is identical to MessageList defined above. If this just renames it, please remove MessageList (it's only used here).